src/maps/Messages.ts

```diff
@@ -126,6 +127,15 @@ export class Message {
     ack() {
         this.channel?.ack(this);
     }
+
+    /**
+     * Reply to Message
+     */
+
+    async reply(data: string | (Omit<Route<'POST', '/channels/id/messages'>["data"], 'nonce'> & { nonce?: string }), mention = true) {
+        let obj = typeof data === 'string' ? { content: data } : data;
+        this.channel?.sendMessage({ ...obj, replies: [{ id: this._id, mention }] })
```

Comment:
Please either add a `return await` statement here, or remove `async` from the function and just add `return`.

Reply:
what it should return? or its supposed to be only `return`

Reply:
either:
```js
async reply(..) {

return await this.channel.sendMessage
}
```
or:
```js
reply(..) {

return this.channel.sendMessage
}
```